### ISSUE TITLE
Version numbers identified by variables aren't handled properly

### ISSUE THREAD

**Issue by pmonks** [deps-app/versions#7]
I occasionally use a variable to define a version number for a set of related libraries (usually raw Java libs that all need to be pinned to the same version).

[Here's an example.](https://github.com/symphonyoss/bot-unfurl/blob/master/project.clj#L18)

Unfortunately versions doesn't seem to handle that properly, and reports out of date dependencies

[Here's the output for the same example.](https://versions.deps.co/symphonyoss/bot-unfurl)

**Comment by pmonks** [comment]
Note: may be one specific symptom of the more general issue described in #5.

**Comment by danielcompton** [comment]
Yeah it's a bit of a problem, because this is a common thing to want to do. I've been thinking maybe about creating a lein and boot plugin that people can run in CI, which would send all of the dependencies in the project on each build. That would let me handle any arbitrarily complex scheme that people do with lein and boot. The other option is that I spin up containers somewhere and evaluate the project myself to get their versions.

**Comment by dotemacs** [comment]
I was just thinking about how this could be done in a container, just for leiningen related projects:

```
wget  https://url-to/project.clj 
lein pprint > result.clj
```

Then parse the result.clj file...
